docs/content/docs/dev/datastream/project-configuration.md

```diff
@@ -372,19 +358,18 @@ dependencies {
     // Compile-time dependencies that should NOT be part of the
     // shadow jar and are provided in the lib folder of Flink
     // --------------------------------------------------------------
-    compile "org.apache.flink:flink-streaming-java:${flinkVersion}"
-    compile "org.apache.flink:flink-clients:${flinkVersion}"
+    implementation "org.apache.flink:flink-java:${flinkVersion}"
```

Comment:
Why are you using a different set of dependencies than the maven quickstarts (flink-java instead of flink-clients)? Note that flink-clients is require to run things in the IDE.

Reply:
Thx, I've fixed that.